content/governance.md

```diff
@@ -70,11 +70,13 @@ The current team members are:
 * Tobias Schmidt ([SoundCloud](https://soundcloud.com/))
 * Tom Wilkie ([Grafana Labs](https://grafana.com/))
 
-The ex-members who are publicly listed are:
+Previous members:
 
 * Max Inden
 * Stuart Nelson
 
+_Note that Prometheus has received significant contributions from a number of unlisted individuals before this governance document and thus formal team membership was created._
```

Comment:
If Matt gives us his agreement, we could add something like: "This includes one of the founders of the project, Matt T. Proud."

Reply:
This is really nice, well put :+1: